editor/common/src/EditorProps.ts

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Editing
+ */
+
+/** @internal */
+export const editCommandApi = {
+  start: "imodeljs.editcmd.start",
+  call: "imodeljs.editcmd.call",
+};
+
+/** @internal */
+export type CommandError =
+  "CommandNotFound" |
+  "Exception" |
+  "MethodNotFound" |
+  "NoActiveCommand";
+
+/** @internal */
+export interface CommandResult<T> {
+  error?: CommandError;
+  result?: T;
+}
+
+/** @internal */
+export interface PingResult {
+  commandId?: string;
+  version?: string;
+  [propName: string]: any;
+}
+
+/** @beta */
+export interface CommandMethodProps<T> {

```

Comment:
Is there some enforcement in the RPC layer that `args` is a POD javascript value?

Reply:
Not at runtime. There was a custom tslint rule that enforced it at compile time. That may be disabled now with eslint though?

Reply:
This api doesn't use our RPC layer. Instead it uses the Electron IPC directly. Their documentation says:

Arguments will be serialized with the [Structured Clone Algorithm,](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm)

Reply:
Probably should document that and note that objects must be handled with care (e.g., no `instanceof`, no functions, etc).